src/common/main.lua

```diff
@@ -90,6 +90,7 @@ function ant.init()
 		local recipient = msg.Tags.Recipient
 		if msg.From ~= Owner then
 			utils.Send(msg, {
+				Target = msg.From,
```

Comment:
Would it make sense to set this in utils.Send automatically if not present rather than making it the responsibility of the caller?

Reply:
Yes it would make sense to add it there - technically its not needed for msg.reply, and we only add this because of ao.send

If we don't, we actually send an invalid message if msg.reply is not present.

Im actually wondering if we even need to wrap ao.send like this, because we KNOW what AOS module we are building. This wrapping pattern is only for backwards compatibility with older AOS process, and with our reassign-to-evolve pattern it will be a new module of our choice anyways.

Also we may not always want to "reply". Sometimes we just want to send. Using reply probably doesnt hurt, but IDK what future integrations may be influenced with these extra tags.

Reply:
and in fact if my suspicions are correct, in that adding function calls is messing with stack memory, then we wanna be careful about that.